server/clearAllData.mjs

```diff
@@ -3,29 +3,39 @@ import redis from "redis";
 import * as path from "path";
 import * as fs from "fs";
 const resultEnv = dotenv.config()
-const client = redis.createClient();
+const redisPrefix = resultEnv.parsed.REDIS_PREFIX
+const client = redis.createClient({prefix: redisPrefix});
 
 
-client.on('connect', () => {
-  let pathToDb = path.resolve(process.env.PATH_TO_CKB_DB)
```

Comment:
Again, this should be configured with a flag rather than removing this functionality as it's very useful for testing in certain circumstances.